src/Razor/src/Microsoft.VisualStudio.RazorExtension/AssemblyBindingRedirects.cs

```diff
@@ -9,60 +9,60 @@
     PublicKeyToken = "adb9793829ddae60",
     OldVersionLowerBound = "0.0.0.0",
     OldVersionUpperBound = "6.0.0.0",
```

Comment:
Should `OldVersionUpperBound` be updated to `7.0.0.0` as well? I think I've seen that pattern to ensure that any `6.*.*.*` versions are redirected. Double-check though, because my memory is fuzzy.

Reply:
I'm testing the runtime a bit to check for any runtime exceptions I can find due to this update